File: Baitap1/Complete/Python/revert_list.py
def rev_list(L):
	"""
	input: L, a list
	Modifies L such that its elements are in reverse order
	returns: nothing
	"""
	for index in range(len(L)//2):
		j = len(L) - index - 1
		temp = L[index]
		L[index] = L[j]
		L[j] = temp

L = [1, 2, 3, 4]
rev_list(L)
print(L)

File: Baitap1/Complete/Java/Revert_List.java
class Revert_List{
    public static void main(String[] args) {
        int[] a = {1 , 2, 3, 4};
        int size_a = 4 ;
        for(int index = 0 ; index <= (size_a-1)/2 ; index++){
            int index_opposite = size_a - 1 - index;
            int temp = a[index];
            a[index] = a[index_opposite];
            a[index_opposite] = temp;
        }
        for(int i = 0 ; i < size_a ; i++){
            System.out.printf(a[i] + "  ");
        }
    }
}